name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/CreditSuisseAGPDFExtractor.java

```diff
@@ -173,7 +173,12 @@ private void addDividendeTransaction()
                 // Coupon-Verfall 10.04.2021
                 .section("date")
                 .match("^Coupon-Verfall (?<date>\\d+.\\d+.\\d{4})$")
-                .assign((t, v) -> t.setDateTime(asDate(v.get("date"))))
+                .assign((t, v) -> t.setDateTime(asDate(v.get("date"))))                
+
+                // Ex-Dividenden-Tag 29.11.2016
```

Comment:
// // Ex-Dividenden-Tag 29.11.2016
not the same for regEx
`^Ex-Datum (?<exDate>\\d+.\\d+.\\d{4})$`